.github/workflows/migrations.yml

```
@@ -81,7 +81,7 @@ jobs:
         id: file
         run: |
           echo 'added<<EOF' >> "$GITHUB_OUTPUT"
-          git diff --diff-filter=A --name-only origin/master HEAD | grep 'src/sentry/migrations/' >> "$GITHUB_OUTPUT"
+          git diff --diff-filter=A --name-only origin/master HEAD | grep -E 'src/[^\/]*/migrations/' >> "$GITHUB_OUTPUT"
```

can use git's filtering directly

```suggestion
          git diff --diff-filter=A --name-only origin/master HEAD -- 'src/sentry/*/migrations/' 'src/sentry/migrations/' >> "$GITHUB_OUTPUT"
```